lib/components/artwork_grids/__tests__/infinite-scroll-grid-tests.js

```diff
@@ -0,0 +1,40 @@
+import 'react-native'
+
+import React from 'react'
+import { renderWithLayout } from '../../../tests/render_with_layout'
+
+import InfiniteScrollArtworksGrid from '../infinite_scroll_grid'
+
+jest.mock('../../../metaphysics.js', () => 'Metaphysics')
+jest.mock('../../opaque_image_view.js', () => 'AROpaqueImageView')
+jest.mock('../../spinner.js', () => 'ARSpinner')
+
+it('renders properly', () => {
+  const artworks = [ artwork(), artwork(), artwork() ]
+
+  const grid = renderWithLayout(<InfiniteScrollArtworksGrid artworks={artworks} />, { width: 768})
+  expect(grid).toMatchSnapshot()
+})
+
+var artwork = () => {
```

Comment:
hrm, ideally our linters be complaining that this should be a `const`, it's not something being mutated